src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -748,6 +751,8 @@
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --aad-client-app-id {APP_ID} --aad-client-app-secret {APP_SECRET} --aad-tenant-id {TENANT_ID} --compute-count 5
   - name: Create an Openshift cluster using a custom vnet
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --vnet-peer "/subscriptions/0000000-0000-0000-0000-000000000000/resourceGroups/openshift-vnet/providers/Microsoft.Network/virtualNetworks/test"
+  - name: Create an Openshift cluster with Log Analytics monitoring enabled
+    text: az openshift create -g MyResourceGroup -n MyManagedCluster --workspace-resource-id {WORKSPACE_ID}
 """
```

Comment:
From reviewing the code, I understand that, customer can only add the monitoring as part of cluster creation.  what about the scenario to add monitoring for existing ARO cluster?

Reply:
I understand the RP will support changing the value - @troy0820 the `az` client will need to as well.

Reply:
That is correct.  I think that scenario to add monitoring to an existing cluster is valid and I am open up to feedback on how AKS does this.

Reply:
I have tested on update if you add the monitor profile, or have one and enable it, it will add the agent to the cluster and logs will flow. Simarily you can change the workspace and the agents will be updated and logs will flow to the new namespace